p2panda-auth/src/traits/group_store.rs

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use std::error::Error;
+
+use super::IdentityHandle;
+
+/// API for global group store.
+pub trait GroupStore<ID>
+where
+    ID: IdentityHandle,
+{
+    type Group;
```

Comment:
Not sure if this needs to be generic. This will almost always be a `GroupState` or not?

Reply:
Yes, you're right. If we had a new AuthGroup implementation it would likely have it's own state store. Nice.